exercises/concept/take-a-number/.docs/instructions.md

```diff
@@ -15,12 +15,15 @@ Note that each time you run this code, the PID may be different.
 
 ## 2. Report the machine state
 
-Modify the machine so that it can receive `{:report_state, sender_pid}` messages. It should send its current state (the last given out ticket number) to `sender_pid` and then wait for more messages.
+Modify the machine so that the newly spawned process is ready to receive messages (start a _receive loop_) with an initial state of `0`. It should be able to receive `{:report_state, sender_pid}` messages. As a response to those messages, it should send its current state (the last given out ticket number) to `sender_pid` and then wait for more messages.
 
 ```elixir
 machine_pid = TakeANumber.start()
+
+# a client sending a message to the machine
 send(machine_pid, {:report_state, self()})
 
+# a client receiving a message from the machine
 receive do
   msg -> msg
 end
```

Comment:
Are those comments helpful? I was staring at this snippet and I was afraid that people might not understand that this code is an example code of how to test the implementation of the machine, and not the code that should go into the implementation of the machine 🤔

Reply:
I see your concern, but that's usually what those code snippets do.
Maybe if you wanted to be even more explicit, you could frame those as tests? 
```elixir
  test "reports its own state" do
    pid = TakeANumber.start()
    send(pid, {:report_state, self()})
    assert_receive 0
  end
```
but that might not help so much. 
I think adding the comments is fine for now.